styles/pypi/catppuccin.user.css

```diff
@@ -16,6 +16,8 @@
 ==/UserStyle== */
 
 @-moz-document domain('pypi.org') {
+  @import url("https://unpkg.com/@catppuccin/highlightjs@0.1.2/css/catppuccin.variables.css");
```

Comment:
Oh no that was just an example. PyPI uses Pygments, not HighlightJS, so this won't work.